src/shared/fetch-with-csrf-token.js

```diff
@@ -12,11 +12,20 @@ export function getCookie(name) {
   return decodeURIComponent(token[0].split('=')[1])
 }
 
-export default function fetchWithCSRFToken(url, otherParts) {
+// for POST of files, primarily
+export default function fetchWithCSRFToken(url, otherParts, headers = {}) {
   const csrfToken = getCookie('csrftoken')
-  const headers = {
-    'Content-Type': 'application/json',
+  const defaultHeaders = {
+    // 'Content-Type': 'application/json', // THIS NEEDS TO BE REPLACED NOW!
```

Comment:
yup this needs fixin' before landing :)

Reply:
I believe I actually separated this out already, so we can delete this commented-out line. See the function `fetchWithCSRFTokenAndJSONContent` below.